anomalib/deploy/inferencers/openvino_inference.py

```diff
@@ -141,9 +149,30 @@ def post_process(
         anomaly_map = predictions.squeeze()
         pred_score = anomaly_map.reshape(-1).max()
 
+        # Common practice in anomaly detection is to assign anomalous
+        # label to the prediction if the prediction score is greater
+        # than the image threshold.
```

Comment:
This would bring up another discussion. Do you think we should check if they are in meta_data?

For example,

```
if "image_threshold" in meta_data:
    pred_label = pred_score >= meta_data["image_threshold"]
```

We cannot calculate the pred-label if we don't have this. The question is whether we should have this if statement here, and for the rest of the computation?

Reply:
I think the default threshold is 0.5 so if we don't have `image_threshold` in meta_data then we can throw a warning and reduce the if statement to `pred_label = pred_score >= meta_data.get("image_threshold", 0.5)`

Reply:
As long as we throw a warning it should be fine. Because the users need to be made aware that there is a problem with the metadata file.